templates/services/common/cf/envvars.yml

```diff
@@ -2,18 +2,18 @@
 # This lets customers have access to, for example, their LB endpoint - which they'd
 # have no way of otherwise determining.
 Environment:
-- Name: ECS_CLI_PROJECT_NAME
-  Value: !Sub '${ProjectName}'
-- Name: ECS_SVC_DISCOVERY_ENDPOINT
-  Value: !Sub '${ProjectName}.local'
+- Name: ECS_CLI_APPLICATION_NAME
```

Comment:
`COPILOT_APPLICATION_NAME` and so on for these :P

Reply:
😅